src/ethers.ts

```diff
@@ -914,8 +971,8 @@ export async function launchPolygon() {
                 console.debug(`Querying native logs from ${startHeight} to ${endHeight} = ${endHeight - startHeight}`);
 
                 let [logsIn, logsOut/* , metaTxs */] = await Promise.all([ // eslint-disable-line no-await-in-loop
-                    client.usdcToken.queryFilter(filterIncoming, startHeight, endHeight),
-                    client.usdcToken.queryFilter(filterOutgoing, startHeight, endHeight),
+                    safeQueryFilter(client.usdcBridgedToken, filterIncoming, startHeight, endHeight),
+                    safeQueryFilter(client.usdcBridgedToken, filterOutgoing, startHeight, endHeight),
```

Comment:
Copy-past error probably.
```suggestion
                    safeQueryFilter(client.usdcToken, filterIncoming, startHeight, endHeight),
                    safeQueryFilter(client.usdcToken, filterOutgoing, startHeight, endHeight),
```